### ISSUE TITLE
Adding rotation speed on pointOfView does not center to the right point

### ISSUE THREAD

**Issue by AldoS** [vasturiano/react-globe.gl#64]
**Describe the bug**
When I add a rotation speed in the point of view the map doesn't center to the selected point

**To Reproduce**
1. Add some points to different locations on the globe
2. On an `onCustomLayerClick` change the pointOfView with any ROTATION_SPEED > 0
`globeEl.current.pointOfView({...}, ROTATION_SPEED);` 
3. If ROTATION_SPEED is 0 the map center to the right point
3. If ROTATION_SPEED > 0 the map does not center to the right point

**Expected behavior**
The map should center to the selected point.

**Desktop (please complete the following information):**
 - OS: macOS High Sierra
 - Browser: chrome

**Comment by vasturiano** [comment]
@AldoS thanks for reaching out.

I'm unable to reproduce the issue. Could you make a simple example on [codesandbox]©desandbox.io) that shows that incorrect behavior?

**Comment by AldoS** [comment]
Hi @vasturiano, here is a simple example: https://codesandbox.io/s/confident-moon-q0vj9

You can click through the labels on the map and you should see the issue I am mentioning.
As I said the issue only appears when the ROTATION_SPEED is > 0.

Thank you

**Comment by vasturiano** [comment]
@AldoS thanks for making the example.

After some digging I found the cause of the issue. You are passing `lat` and `lng` coordinates as strings. These fields should be numeric. When the animation time is `0` it works fine (by luck), because the camera coordinates are directly assigned the final values. But with an animation, the tween control has to interpolate between the two values. If the inputs are strings it will incorrectly interpolate the coordinates, causing the effect you described.

So, if you switch those values to numeric it should start working as expected.

I've also patched the interpolator to accept strings for the sake of convenience, so if you pull the latest version of the module it should also start working.
